pkg/controller/populators/import-populator.go

```diff
@@ -210,15 +204,23 @@ func updateImportSourceAnnotation(pvc, pvcPrime *corev1.PersistentVolumeClaim) {
 	pvc.Annotations[cc.AnnSource] = cc.GetSource(pvcPrime)
 }
 
+func (r *ImportPopulatorReconciler) updateImportAnnotations(pvc, pvcPrime *corev1.PersistentVolumeClaim) {
+	phase := pvcPrime.Annotations[cc.AnnPodPhase]
+	if err := r.updateImportProgress(phase, pvc, pvcPrime); err != nil {
+		//TODO: currently I don't return err in case of failing to report progress, opinion?
+		// Also this will print error every 2 secs
+		klog.Errorf("Failed to update import progress for pvc %s/%s", pvc.Namespace, pvc.Name)
```

Comment:
why klog and not the logr.Logger is my first question?

If we requeue every 2secs I guess it isn't a big deal to not handle errors but maybe would be nice

Reply:
I didn't want to return an error cause I think we shouldn't prevent the phases to be updated just because there might be an issue with the progress reporting.
regarding the klog, I'll fix